tests/sentry/api/serializers/test_app_platform_event.py

```diff
@@ -2,6 +2,7 @@
 
 from sentry.api.serializers import AppPlatformEvent
 from sentry.testutils.cases import TestCase
+from sentry.utils import json
```

Comment:
Can you move this to line 22, and can you use `import json` instead of this file, and add a comment to why we have this?